autoscheduler/frontend/src/tests/ui/CourseSelectCard.test.tsx

```diff
@@ -561,6 +562,8 @@ describe('Course Select Card UI', () => {
         const { getByText } = render(
           <Provider store={store}><CourseSelectCard id={0} /></Provider>,
         );
+        // Stop course card from loading
+        store.dispatch<any>(updateCourseCard(0, { loading: false }));
```

Comment:
It would attempt to fetch saved sections in `<CourseSelectColumn>`, find an empty array, and dispatch `replaceCourseCards` with an empty array which would set the initial course card to stop loading. Since these tests don't actually have the `<CourseSelectColumn>` component to tell the card to stop loading, this is the only way I could think of to do this other than overwriting the default redux store.

Reply:
Okay, makes sense